pages/1.10/deploying-services/uninstall/index.md

```diff
@@ -8,10 +8,9 @@ excerpt: Uninstalling DC/OS services from the CLI
 enterprise: false
 ---
 
+Most services can be uninstalled from the CLI or the DC/OS web interface. If a Universe service has any reserved resources that could not be be cleaned up by the normal uninstall process, you may also need to run the framework cleaner script. The [framework cleaner script](#framework-cleaner) removes the service instance from ZooKeeper, along with any data associated with it.
```

Comment:
As a user, how do I qualify `Most services can be uninstalled`? How can I say that the service I am trying to uninstall qualifies as the "most"?

Reply:
By "most", I mean "not all". This covers those services which can only be uninstalled from the CLI, or services which can only be uninstalled from the web interface, as well as those which can be uninstalled by either method. I'm not sure I understand your second sentence. "The most" does not have the same meaning as "most"; "the most" is a superlative like "the greatest", but "most" just means "many" or "the majority". I am unsure which meaning you are referring to.